specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2023-05-01-preview/definitions.json

```diff
@@ -1957,6 +1957,24 @@
         "circuitBreaker": {
           "$ref": "#/definitions/BackendCircuitBreaker",
           "description": "Backend Circuit Breaker Configuration"
+        },
+        "pool": {
+          "allOf": [
+            {
+              "$ref": "#/definitions/BackendPool"
+            },
+            {
+              "description": "Backend Pool Properties"
+            }
+          ]
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "single",
+            "pool"
```

Comment:
instead of using enums use `x-ms-enum`

Reply:
https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#use-extensible-enums